account_credit_control/__manifest__.py

```diff
@@ -19,33 +19,43 @@
 #
 ##############################################################################
 {'name': 'Account Credit Control',
- 'version': '8.0.0.3.0',
- 'author': "Camptocamp,Odoo Community Association (OCA)",
+ 'version': '10.0.1.0.0',
+ 'author': "Camptocamp,Odoo Community Association (OCA),Okia",
```

Comment:
You must separate authors or you get a linter error:

Correct is  "Camptocamp, Odoo Community Association (OCA), Okia",